create view

kawa18sima opened the pull request:
# WHAT
Mark up the chat screen.

# WHY
View is necessary.

Commit:
create pull request

Commit:
Change from application.css to application.scss

Changing extension from application.css to application.scss
Change content

Commit:
Create _reset.scss

Create reset file of YUI 3

Commit:
'Font-awesome-rails' added to Gemfile

Added to display icon in view

Commit:
Bundle install

To install font-awesome

Commit:
Edit Application.scss

Added description to load 'font-awesome' in application.scss

Commit:
Rails g controller messages

Create the messages message controller

Commit:
Markup chat screen

Mark up the chat screen with Haml Sass

kawa18sima commented:
<img width="1440" alt="2017-04-17 22 06 05" src="https://cloud.githubusercontent.com/assets/24731933/25089900/47579450-23bb-11e7-9e8c-9db994ccc3e1.png">

遅くなってすみません

kinshotomoya commented on app/assets/stylesheets/application.scss:
```diff
@@ -0,0 +1 @@
+@import "./reset";
```
改行しましょう！

kinshotomoya commented on app/assets/stylesheets/_reset.scss:
```diff
@@ -0,0 +1,126 @@
+    color:#000;
+}
+/* YUI CSS Detection Stamp */
+#yui3-css-stamp.cssreset { display: none; }
```
改行しましょう！

kinshotomoya commented on app/assets/stylesheets/application.scss:
```diff
@@ -1 +1,2 @@
-@import "./reset";
\ No newline at end of file
+@import "./reset";
+@import "font-awesome";
```
改行しましょう！

kinshotomoya commented on app/assets/stylesheets/application.scss:
```diff
@@ -1,2 +1,5 @@
\ No newline at end of file
+@import "mixin/*";
+@import "config/*";
+@import "messages";
```
改行しましょう！

kinshotomoya commented on app/assets/stylesheets/config/_variable.scss:
```diff
@@ -0,0 +1,10 @@
+$title__black:#333;
+$light-black:#999;
+$text__black:#434a54;
+$dark-gray:#808080;
```
改行しましょう！

kinshotomoya commented on app/assets/stylesheets/application.scss:
```diff
@@ -1,2 +1,5 @@
 @import "./reset";
-@import "font-awesome";
\ No newline at end of file
+@import "mixin/*";
```
*いる？？

kinshotomoya commented on app/assets/stylesheets/messages.scss:
```diff
@@ -1,3 +1,143 @@
+}
+
+.notification{
+  color:white;
```
せっかく_valiable.scssで書いてるのに、、！

kinshotomoya commented on app/assets/stylesheets/messages.scss:
```diff
@@ -1,3 +1,143 @@
+  }
+  &__icon{
+    float:right;
+    color:white;
```
ここもです！

kinshotomoya commented on app/views/messages/index.html.haml:
```diff
@@ -1,2 +1,48 @@
+      #li.messages__text
+        難しい
+    .post-space
+      = form_for(@messages) do |f|
```
もうform_forで書いてる、すごい！
@messagesどこにも定義してないからエラー出ませんでした？？
まだ、フロントなのでform_forで無理に書かなくてもいいですよ！

kinshotomoya reviewed (commented):
遅くなってすっません、
難しいですよね、僕も新メンターの時ここが一番苦戦しました笑
いい感じにできているので、あとは細かいところを直していきましょう！

Commit:
HTML CSS editing

・Add the last line break of the file
・Apply variables used in _variable.scss
・* -> File name

kawa18sima commented:
変更しました。
ファイルの最後の改行を忘れがちなので気をつけます。

・ファイルの最後を改行する
・_variable.scssで使用される変数の適用
・application.scssの「*」 - >ファイル名

kinshotomoya commented on app/assets/stylesheets/application.scss:
```diff
@@ -1,5 +1,5 @@
-@import "config/*";
+@import "mixin/clearfix";
+@import "config/variable";
 @import "messages";
```
改行しましょう！！！

kinshotomoya reviewed (commented):
一点だけ修正してマージしましょう！！！
次も頑張りましょう！！！！！！！！！！！！

Commit:
HTML CSS editing

・Add the last line break of the file

Commit:
Syntax error resolution

Delete "/"
